st2actions/Makefile

```diff
@@ -27,13 +27,6 @@ rpm:
 	cp packaging/rpm/$(COMPONENTS).spec $(RPM_SPECS_DIR)/
 	cd $(RPM_SPECS_DIR) && rpmbuild --clean --rmsource -ba $(COMPONENTS).spec
 
-.PHONY: rhel-rpm
```

Comment:
Just FYI the rpm/deb logic in the Makefiles of `st2` repository is really outdated and unused code we're guilty to remove before.
All packaging happens in https://github.com/stackstorm/st2-packages